Prog/Eviction Group/DC-evic-analysis-need.R

```diff
@@ -4,19 +4,17 @@
 # Author: Elizabeth Burton
 # Created: 1/16/2024
 
-# Description: Estimate # households in DC that need housing assistance. For eviction co-leader group 
-# housing assistance analysis
+# Description: Estimate # households in DC that need housing assistance. For eviction co-leader group housing assistance analysis
 ##################
 
 # Load libraries and functions
 library(tidyverse)
 library(tidycensus)
-library(skimr)
-library(data.table)
 library(openxlsx)
+library(Hmisc)
 
 # Household income bins function
-source("C:/Users/eburton/Documents/GitHub/Requests/Prog/Eviction Group/DC-HH-inc-cat.R")
+source("//sas1/dcdata/Libraries/Requests/Prog/Eviction Group/DC-HH-inc-cat.R") # script where household income limits created
 
```

Comment:
DC PUMS data has 33,609 observations and 29 variables. Using person-level weights throughout (`WGTP`).